benefits/static/css/admin/styles.css

```diff
@@ -261,3 +257,25 @@ iframe.card-collection {
   font-size: var(--font-size-16px);
   line-height: 1;
 }
+
+/* Login Page: Django messages style overrides */
+
+ul.messagelist {
```

Comment:
This feels like a pretty generic selector, that could grab more than just `ul.messagelist` on the login page. Maybe that is OK?

Reply:
Yeah, you're absolutely right. Django messages could appear on many other admin pages and we wouldn't want this styling for those.